### ISSUE TITLE
Framework Request: StoreKit

### ISSUE THREAD

**Issue by AnalogJ** [progrium/darwinkit#235]
It would be great if we could use macdriver to manage App Store subscriptions

**Comment by programmingkidx** [comment]
You want the StoreKit framework added to ~~MacDriver~~ DarwinKit?

**Comment by AnalogJ** [comment]
Yep. If I understand how StoreKit works, it would allow me to verify subscription statuses within my Go application.
